test_bot/config.py

```diff
@@ -0,0 +1,25 @@
+import os
+
+try:
+    import dotenv
+except ModuleNotFoundError:
+    print("Not loading .env")
+else:
+    dotenv.load_dotenv()
```

Comment:
Would rather not add this at this time.

Reply:
I'll be deferring this to a second pr to add an argparse argument parser interface for the test bot so this can be set at runtime, and other options can be configured.

Reply:
What other options would have to be configured with commandline arguments, that can't already be set through env vars? I don't really see the point in adding argparse if most/all of the configuration is already being done through getenv.

Reply:
futureproof for other arguments.